README.md

```diff
@@ -126,7 +131,7 @@ Activity|Command
 ---|---
 Installation|`$ docker pull mverkerk/timings:{version}`
 Startup|`$ docker run \`<br>`-d -v {path to config}:/src/.config.js \`<br>`-p {VM_port}:{Host_port} \`<br>`mverkerk/timings:{version}`
-Upgrade|Just point at the latest version or use `mverkerk/timings:latest` in the startup command.<br>You can find the latest version here: https://hub.docker.com/r/mverkerk/timings/tags/
+Upgrade|Just point at the latest version or use `mverkerk/timings:latest` in the startup command.<br>You can find the latest version here: [https://hub.docker.com/r/mverkerk/timings/tags/](https://hub.docker.com/r/mverkerk/timings/tags/)
```

Comment:
We have a GoDaddy docker hub account -- do you want to switch to using that?

Reply:
yes, totally! tell me how and i'll get it done!